packages/notifications/vite.config.js

```diff
@@ -0,0 +1,12 @@
+import { viteConfig as defineBaseConfig } from '@terra-ui-packages/build-utils';
+import { defineConfig } from 'vite';
+
+export default defineConfig((args) => {
+  const baseConfig = defineBaseConfig(args)
+  return {
+    ...baseConfig,
+    plugins: [
+      ...baseConfig.plugins,
+    ]
+  }
+});
```

Comment:
Since there's no package specific configuration here, this could be simplified to:

```suggestion
export { viteConfig as default } from '@terra-ui-packages/build-utils';
```

Reply:
hmmm.... kinda like having the config with template as is, in that it is ready for obvious extension.

Reply:
Fair enough. If I'm being nitpicky, I think I'd still suggest removing the object spread stuff and just returning `baseConfig`.

Reply:
ok. meet ya in the middle... kept a one-liner to pass-thru the base config

`export default defineConfig((args) => defineBaseConfig(args));`